app/main.py

```diff
@@ -1 +1,48 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
```

Comment:
Sorry, forgot to mention earlier - you should also add here empty line before __init__, or remove it in all other classes - stick to the same code style everywhere